dependencies.yaml

```diff
@@ -113,6 +119,22 @@ dependencies:
               cuda: "11.8"
             packages:
               - nvcc_linux-aarch64=11.8
+  docs:
+    common:
+      - output_types: [conda]
+        packages:
+          - doxygen
+          - graphviz
+          - ipython
+          - nbsphinx
+          - numpydoc
+          - pydata-sphinx-theme
+          - recommonmark
+          - sphinx_rtd_theme
```

Comment:
```suggestion
```

We know from the [contents of conf.py](https://github.com/AyodeAwe/cugraph/blob/10c37b3932a5b1bda871bcabaa6d3cf2daf27d95/docs/cugraph/source/conf.py#L108) that `cugraph` uses the `pydata-sphinx-theme`.

Therefore we don't also need to install the `sphinx_rtd_theme`.